includes/admin/class-wc-stripe-settings-controller.php

```diff
@@ -62,11 +62,12 @@ public function admin_scripts() {
 		}
 
 		$params = [
-			'time'             => time(),
-			'i18n_out_of_sync' => wp_kses(
+			'time'              => time(),
+			'i18n_out_of_sync'  => wp_kses(
 				__( '<strong>Warning:</strong> your site\'s time does not match the time on your browser and may be incorrect. Some payment methods depend on webhook verification and verifying webhooks with a signing secret depends on your site\'s time being correct, so please check your site\'s time before setting a webhook secret. You may need to contact your site\'s hosting provider to correct the site\'s time.', 'woocommerce-gateway-stripe' ),
 				[ 'strong' => [] ]
 			),
+			'upe_setting_value' => isset( $gateway_settings['upe_checkout_experience_enabled'] ) ? 'no' : $gateway_settings['upe_checkout_experience_enabled'],
 		];
 		wp_localize_script( 'woocommerce_stripe_admin', 'wc_stripe_settings_params', $params );
```

Comment:
I'm getting `woocommerce_stripe_admin is not defined` in the console. It seems like `wp_localize_script()` is not injecting the `woocommerce_stripe_admin` object even though it's returning `true` 🤔

Reply:
> Additionally, `wp_localize_script()` has to be called after `wp_enqueue_script()`

I didn't see information about this online - it looks like `wp_localize_script()` can be called after `wp_register_script()`.

Thanks for pointing out the error, fixed in dc1f7ce 👍

Reply:
I saw the error and glossed over that the object name was just wrong. 🤦 

>wp_localize_script() MUST be called after the script has been registered using wp_register_script() or wp_enqueue_script().

You're right, it just has to be after _one_ of those two. 

Thanks for the fix!